index2.html

```diff
@@ -0,0 +1,213 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="styles2.css">
+    <title>Concept pagina</title>
+</head>
+
+<body>
+<header class="outer-container">
+
+    <nav class="navbar">
+        <ul>
+            <li><a href="#rolling-stone-title">case studies</a></li>
+            <li><a href="#webdesign-title">services</a></li>
+            <li><a href="#footer-text">contact us</a></li>
+        </ul>
+    </nav>
+
+    <div class="hero-section">
+        <h1>Your website is an investment
+            in the succes of your business.</h1>
+        <p>We are here to help you succeed.</p>
+        <button class="cta.btn" type="button">Lets work together</button>
+    </div>
+
+
+    <img src="assets/squiggle_line_stripe.svg" alt="My Happy SVG" class="swirl">
+    <div class="skewed">skewed</div>
```

Comment:
Je hebt een div gemaakt met hierin de tekst “skewed2”. Deze heb jij wel ontzettend slim weggetoverd, maar ik weet beter! :p Als ik nu alsnog tekst selecteer zie ik er “skewed” en “skewed2” staan, tevens loopt het niet goed schuin af onderaan de pagina. Een mogelijke oplossing is het maken van een herbruikbare classname i.c.m <div> (denk aan Elwyn zijn reusables!).

Als je een div maakt met <div class=”skewed”></div> en in de css de class skewed aanspreekt, is die class sowieso herbruikbaar. Je kan dan eerst de algemene instellingen van skewed instellen (vergeet de display: block) niet en als je dat hebt gedaan kan je custom properties instellen.

Reply:
ik zat inderdaad te klooien met het exact uitlijnen van rand tot rand. dat doe ik dus met display block ipv flex?

ik zal de tekst weghalen in het vervolg ipv verstoppen haha!

Reply:
Klopt inderdaad! We willen namelijk dat de skewed statisch is!